data_steward/cdr_cleaner/cleaning_rules/base_cleaning_rule.py

```diff
@@ -322,6 +318,21 @@ def get_sandbox_dataset_id(self):
         """
         return self._sandbox_dataset_id
 
+    @abstractmethod
```

Comment:
this getter would not be abstract anymore. it would exist when the extending class is instantiated.